### ISSUE TITLE
Fix things after 1.0 release

### ISSUE THREAD

**Issue by zsunberg** [JuliaPOMDP/POMDPs.jl#550]
Since I created v1.0, a few things have broken. Most importantly, the docs don't build. This mostly means that we have to update the Project.toml files for all the things needed to build the docs.

**Comment by zsunberg** [comment]
To fix the docs, looks like we need to update:

- [ ] BasicPOMCP
- [ ] DroneSurveillance 
- [ ] MCTS 
- [ ] NativeSARSOP 
- [ ] POMDPGifs 
- [ ] POMDPLinter 
- [ ] POMDPModels 
- [ ] POMDPTools 
- [ ] ParticleFilters 
- [ ] QMDP 
- [ ] QuickPOMDPs 
- [ ] RockSample 
- [ ] TagPOMDPProblem

**Comment by dylan-asmar** [comment]
Fixed RoombaPOMDP which solved the docs issue.

**Comment by zsunberg** [comment]
Now that the docs build, we'll need to update the rest of the packages. The procedure for this is:

1. Add 1 to the compat entries for POMDPs and POMDPTools
2. Increase the PATCH version
3. Register the new patch version

**Comment by zsunberg** [comment]
Here is the list of packages with the ones already completed above:

@dylan-asmar I think that you have already done some of the compat updates, but we should also register new patch versions for all of these because if one of them doesn't use POMDPs v1.0, it will hold everything back.

- [ ] LaserTag
- [ ] ContinuumWorld
- [ ] VDPTag2
- [ ] DiscreteValueIteration
- [ ] LocalApproximationValueIteration
- [ ] GlobalApproximationValueIteration
- [ ] FIB
- [ ] BeliefGridValueIteration
- [ ] SARSOP
- [ ] ParticleFilterTrees
- [ ] ARDESPOT
- [ ] AdaOPS
- [ ] MCVI
- [ ] POMDPSolve
- [ ] IncrementalPruning
- [ ] POMCPOW
- [ ] AEMS
- [ ] PointBasedValueIteration
- [ ] TabularTDLearning
- [ ] DeepQLearning
- [ ] SymbolicMDPs (Just a PR I suppose, since we don't have control over this)

**Comment by dylan-asmar** [comment]
@zsunberg I think we are close to complete.

- [ContinuumWorld](https://github.com/JuliaPOMDP/ContinuumWorld.jl): I decided not to touch this one until we address https://github.com/JuliaPOMDP/ContinuumWorld.jl/issues/5. Thoughts were the decision on that issue would result in a refactor and then updates to the compatibility would happen during that process.
- [SymbolicMDPs](https://github.com/JuliaPlanners/SymbolicMDPs.jl): Waiting on a merge of the PR (https://github.com/JuliaPlanners/SymbolicMDPs.jl/pull/5)
- [VDPTag2](https://github.com/zsunberg/VDPTag2.jl): Since this is a repo under your account, I didn't know if there were specific things you prefer and thought it was best to leave it up to you.
